README.md

```diff
@@ -164,6 +164,12 @@ MacOS support is new. Please create an issue if something doesn't work.
 
 </details>
 
+<details> <summary>NetBSD</summary>
+
+Additionally, there is support for NetBSD on an experimental
+basis. Packages `clang`, `libLLVM`, `gmake` and `bash` are required.
```

Comment:
I forgot that these are inside `<details>` sections. IMO full instructions would be better (copy/paste from linux section as needed), because it's still easy to ignore the parts that you don't care about.